packages/block-editor-utils/src/controls/Radio.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { RadioControl } from '@wordpress/components';
+import { ControlProps } from '../types/index.js';
+
+function Radio<T extends Record<string, any>>({
+  config,
+  props,
+}: ControlProps<T>) {
+  const onChange = (newContent: string) => {
+    props.setAttributes({ [config.name]: String(newContent) });
+  };
+  return (
+    <RadioControl
+      label={config.label}
```

Comment:
Nicely done Matt! Does this also need the `help` attribute described[ here](https://developer.wordpress.org/block-editor/reference-guides/components/radio-control/#help-string-wpelement)?

Reply:
I think we ca leave help out since we cannot assume a good help label for now.